AutoRaise.mm

```diff
@@ -377,27 +376,27 @@ - (void) readConfig: (int) argc {
 
 - (void) readOriginalConfig {
     // original config files:
-    NSString *delayFile = @"AutoRaise.delay";
-    NSString *warpFile  = @"AutoRaise.warp";
-    
-    if ([self fileExist: delayFile] or [self fileExist: warpFile]) {
-        NSFileHandle *hDelayFile = [NSFileHandle fileHandleForReadingAtPath: [self getFilePath: delayFile]];
+    NSString *delayFilePath = [self getFilePath: @"AutoRaise.delay"];
+    NSString *warpFilePath = [self getFilePath: @"AutoRaise.warp"];
```

Comment:
oh..
it is so obviously when I see that now >.<

Reply:
> it all looks much neater and clearer now.
> I need to train a lot if I want to get better.

I think for a first attempt, this was really good actually.
There wasn't really that much I had to change.

Reply:
Thank you! )